docs/iris/gallery_code/meteorology/plot_lagged_ensemble.py

```diff
@@ -62,10 +60,9 @@ def main():
     # for the purposes of this example, take the last time element of the cube
     last_timestep = surface_temp[:, -1, :, :]
```

Comment:
This might be better using `surface_temp.subset(surface_temp.coord(“time”)[-1])`, as it removes the requirement to know the dimension order.  Bit verbose though.

Reply:
Personally, I've never used `subset`, remarkable, huh...

But the gallery is an ideal place to advertise functionality and patterns for `iris`... so I vote for adding it to the shop window. Go for it :+1: